projects/addon-editor/components/editor-socket/editor-socket.component.less

```diff
@@ -8,6 +8,18 @@
     &._preview-image img {
         cursor: pointer;
     }
+
+    h1,
+    h2,
+    h3,
+    h4,
+    h5,
+    h6 {
+        word-break: break-word;
+        line-height: normal;
+        font-weight: normal;
```

Comment:
Are you sure it's not supposed to be bold?

Reply:
fixed